src/main/scala/org/ergoplatform/nodeView/history/storage/HistoryStorage.scala

```diff
@@ -24,17 +24,20 @@ class HistoryStorage private(indexStore: LDBKVStore, objectsStore: LDBKVStore, c
     with AutoCloseable
     with ScorexEncoding {
 
-  private val headersCache = CacheBuilder.newBuilder()
-    .maximumSize(config.history.headersCacheSize)
-    .build[String, BlockSection]
+  private val headersCache =
+    Caffeine.newBuilder()
+      .maximumSize(config.history.headersCacheSize)
+      .build[String, BlockSection]()
 
-  private val blockSectionsCache = CacheBuilder.newBuilder()
-    .maximumSize(config.history.blockSectionsCacheSize)
-    .build[String, BlockSection]
+  private val blockSectionsCache =
+    Caffeine.newBuilder()
+      .maximumSize(config.history.headersCacheSize)
```

Comment:
why headersCacheSize here ? Looks like a mistake.

Reply:
Uff, I misread the lines